Source/EasyNetQ/QueueConsumerPair.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using EasyNetQ.Consumer;
+using EasyNetQ.Internals;
+using EasyNetQ.Topology;
+
+namespace EasyNetQ
+{
+    public class QueueConsumerPair
+    {
+        #region Factory methods
```

Comment:
Personally I think it make reading code much easier, right opposite to your experience....